src/components/hand-tracking-controls.js

```diff
@@ -217,22 +217,22 @@ module.exports.Component = registerComponent('hand-tracking-controls', {
 
       var distance = indexTipPosition.distanceTo(thumbTipPosition);
 
-      if (distance < 0.01 && this.isPinched === false) {
+      if (distance < 0.015 && this.isPinched === false) {
```

Comment:
We can maybe move this to a variable at the top `var PINCH_DISTANCE = 0.015` . We use [similar pattern in other components](https://github.com/aframevr/aframe/blob/master/src/components/oculus-touch-controls.js#L13)

Reply:
Addressed in the latest commit